core/templates/pages/contributor-dashboard-page/modal-templates/question-suggestion-review-modal.controller.ts

```diff
@@ -102,38 +86,126 @@ angular.module('oppia').controller('QuestionSuggestionReviewModalController', [
       });
     };
 
-    $scope.init = function() {
+    $scope.reviewable = reviewable;
+    $scope.misconceptionsBySkill = misconceptionsBySkill;
+    $scope.currentSuggestionId = suggestionId;
+
+    $scope.currentSuggestion = suggestionIdToContribution[suggestionId];
+    delete suggestionIdToContribution[suggestionId];
+    $scope.remainingContributionIds = Object.keys(
+      suggestionIdToContribution
+    );
+    $scope.remainingContributionIds.reverse();
```

Comment:
Why reverse the list?

Reply:
The opportunities are stored in the list in order (top to bottom). However we treat this list as a stack (for navigation) and operate on the last element of the list. Without reversing the list first,  the navigation will be done not in order.

Reply:
I see. Can you leave a comment explaining this? Maybe even renaming it to `remainingContributionIdStack` could help make this clear.

Reply:
Renamed the variable.